language/move-vm/runtime/src/interpreter.rs

```diff
@@ -1007,10 +990,8 @@ impl Frame {
                             .push(Value::bool(!lhs.equals(&rhs)?))?;
                     }
                     Bytecode::GetTxnSenderAddress => {
-                        cost_strategy.charge_instr(Opcodes::GET_TXN_SENDER)?;
-                        interpreter
-                            .operand_stack
-                            .push(Value::address(interpreter.sender))?;
+                        // GetTxnSenderAddress is deprecated and will be removed soon
+                        return Err(VMStatus::new(StatusCode::UNKNOWN_INVARIANT_VIOLATION_ERROR));
```

Comment:
Could you add `.with_message(...)` to indicate what triggered this error? It will help save a lot of time in case someone actually hits this case and need to figure out the origin of the error.

Reply:
Good call--will  push the comment into the `with_message`